packages/utilities/src/utilities/scrollIntoView/scroll-into-view.scss

```
@@ -0,0 +1,24 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import '@carbon/motion/scss/motion.scss';
+
+:root {
+  --delay: 400ms;
```

To avoid variable name conflict with other CSS libraries and application code:

```suggestion
  --#{$dds-prefix}--delay: 400ms;
```